docs/getting-started-auditor.md

```diff
@@ -148,23 +146,15 @@ During the execution, it may detect inconsistencies between them if there is tam
 
 ## Validate the states of Ledger and Auditor
 
-You can also always validate the states of Ledger and Auditor to see if they are consistent with the following command as usual:
+You can also always validate the states of Ledger and Auditor to see if they are consistent.
+However, validating the states in the Auditor mode uses contract execution; thus, you first need to register [ValidateLedger](https://github.com/scalar-labs/scalardl-java-client-sdk/blob/master/src/main/java/com/scalar/dl/client/contract/ValidateLedger.java) contract as follows.
 
-```
-client/bin/validate-ledger --properties client.properties --asset-id="some_asset"
-```
-
-By default, it checks the states of Ledger and Auditor non-linearizably; i.e., there might be cases where they look inconsistent temporarily.
-To make it linearizable, you can configure the following option and register [ValidateLedger](https://github.com/scalar-labs/scalardl-java-client-sdk/blob/master/src/main/java/com/scalar/dl/client/contract/ValidateLedger.java) contract as follows.
-
-```
-[client.properties]
-scalar.dl.client.auditor.linearizable_validation.enabled=true
-```
-
-```
+```shell
```

Comment:
It's not necessary but I think it would be clear if we also introduce the optional property `scalar.dl.client.auditor.linearizable_validation.contract_id` here.

The users have to use the matched id name configured in the client.properties file.

Reply:
Thank you! Updated in [e7cc2bc](https://github.com/scalar-labs/scalardl/pull/34/commits/e7cc2bc1a9f535e5a11a4eb0117391d90e8d3dad).